src/features/fwiCalculator/FWICalculatorPage.test.tsx

```diff
@@ -34,6 +37,15 @@ it('renders FWI calculator page', async () => {
   fireEvent.click(station1)
 
   expect(selectStationsReducer(store.getState()).stations).toEqual(mockStations)
+
+  //Time range dropdown test
+  expect(getByTestId('time-range-dropdown')).toBeInTheDocument()
+
+  //fireEvent.click(getByTestId('time-range-dropdown'))
```

Comment:
Do we need this code if it's commented out?

Reply:
Yes. I commented it as part of current MVP but its still required for later tasks.